### DISCUSSION TITLE
Lokinet Exit Node

### DISCUSSION THREAD

**Discussion by Zardon703** [oxen-io/lokinet#2203] [Q&A] [answered]
Good day,
Is it possible to set up a Lokinet Exit Node? And if it is, what is the prosses?
This request is mainly to help out with the bandwidth limitation
And yes, I understand the legal implications 

Update: It "seems" this was already answered

**Comment by majestrate** [comment, marked as answer]
see https://github.com/oxen-io/lokinet/blob/dev/docs/exit-setup.md

**Comment by Zardon703** [comment]
> see https://github.com/oxen-io/lokinet/blob/dev/docs/exit-setup.md
Ye, that`s the answer i saw. but thanks tho
